scripts/abins/parameters.py

```diff
@@ -17,22 +16,41 @@
     abins.parameters.instruments['TOSCA']['final_neutron_energy'] = 24.0
 """
 
+import math
+from abins.constants import MILLI_EV_TO_WAVENUMBER
+
 # Instruments constants  #############################
 instruments = {
     'fwhm': 3.0,  # approximate value for the full width at half maximum for Gaussian experimental resolutions
-    'TwoDMap': {
-        'delta_width': 0.1,  # width of narrow Gaussian which approximates Dirac delta
-        },
     'TOSCA': {
-        #    TOSCA parameters for calculating Q^2
+        # TOSCA parameters for calculating Q^2
         'final_neutron_energy': 32.0,  # Final energy on the crystal analyser in cm-1
-        'cos_scattering_angle': math.cos(2.356),  # Angle of the crystal analyser radians
+        'cos_scattering_angle': math.cos(2.356),  # Angle of the crystal analyser radians (NO LONGER USED)
+        # The forward detector angle israther specific as test-data was based on truncated value in radians
```

Comment:
Done. The line above with the unused angle should really come out as well, but this will break some of the advanced parameter validation logic. I intend to visit more seriously that in the near future, so would rather not break tests / replace all the test data over a bit of redunancy that will soon be addressed anyway.